### PR TITLE
add demo config

### PR DESCRIPTION
**Pull request by iguannago**

### CODE REVIEW

File: k8s/demo/common/wa/wa-proto-frontend.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/wa-proto-frontend
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      ingressClass: traefik-no-proxy
+      image: hmctspublic.azurecr.io/wa/wa-proto-frontend:latest
+      ingressHost: wa-prot-frontend.demo.platform.hmcts.net
```
Comment (hannah38) [inline k8s/demo/common/wa/wa-proto-frontend.yaml:25]: Looks like a typo here `wa-prot-frontend`

Comment (iguannago) [reply k8s/demo/common/wa/wa-proto-frontend.yaml]: Well, spot! It's fixed now. Cheers!

File: k8s/demo/common/wa/wa-proto-frontend.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    ref: master
+    path: stable/wa-proto-frontend
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      ingressClass: traefik-no-proxy
+      image: hmctspublic.azurecr.io/wa/wa-proto-frontend:latest
+      ingressHost: wa-proto-frontend.demo.platform.hmcts.net
+      environment:
```
Comment (hannah38) [inline k8s/demo/common/wa/wa-proto-frontend.yaml:26]: One more - you won't need this `environment:` if it's empty

Comment (iguannago) [reply k8s/demo/common/wa/wa-proto-frontend.yaml]: fixed. Cheers!

**Review by hannah38 — APPROVED** [review]

**Pull request merged**